src/assets/styles/base.scss

```diff
@@ -315,5 +309,5 @@ ul.list {
 
 .step-container {
   margin-top: 1rem;
-  padding: 1.25rem;
+  padding: 1.5rem;
```

Comment:
This may throw off some alignment on the review summary pages.. can't recall for sure. 
Either way, good to ensure all the step 1 and summaries are still in alignment 👍

Reply:
I think this class isn't used for any review summary pages. I have spot-checked all pages for all filing types and it looks OK, I think. As you mentioned below, a second (and third) set of eyes would be great.

![image](https://user-images.githubusercontent.com/10002889/153917935-b4eee220-7e62-4c80-872a-677a4d0b9ebe.png)

Reply:
I see LOTS of inconsistent whitespace in different filings/pages but I'm trying to touch as little as possible this time around!